core/src/main/java/org/eclipse/hono/config/ServiceConfigProperties.java

```diff
@@ -324,4 +326,30 @@ public final long getEventLoopBlockedCheckTimeout() {
     public final void setEventLoopBlockedCheckTimeout(final long eventLoopBlockedCheckTimeout) {
         this.eventLoopBlockedCheckTimeout = eventLoopBlockedCheckTimeout;
     }
+
+    /**
+     * Gets the idle timeout.
+     * <p>
+     * The idle timeout is used when data is not received, the connection is timeout and closed.
+     * <p>
+     * The default value is {@code 60} in seconds.
+     *
+     * @return The idle timeout value in seconds.
+     */
+    public int getIdleTimeout() {
+        return idleTimeout;
+    }
+
+    /**
+     * Sets the idle timeout.
+     * <p>
+     * The idle timeout is used when data is not received, the connection is timeout and closed.
+     * <p>
+     * The default value is {@code 60}. The idle time unit is seconds.
+     *
+     * @param idleTimeout The idle timeout.
+     */
+    public void setIdleTimeout(final int idleTimeout) {
```

Comment:
It would be good to check for an illegal timeout value here (i.e. a negative value, as done [here](https://github.com/eclipse-vertx/vert.x/blob/00d6ce29fdc258de7f92bc54b15a237955008365/src/main/java/io/vertx/core/net/TCPSSLOptions.java#L286-L288)).